public/app/features/explore/ErrorBoundary.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+
+export default class ErrorBoundary extends Component<{}, any> {
```

Comment:
wonder if this could be made it a more generic error handler , maybe as a decorator class or HOC that other components can use.  Question is should it use the alert popups as way to show error or some other way

Reply:
I'd keep it as a component for now, since you can make clear which parts are bounded. We can add integration with pop ups later.

Reply:
Is it not possible to use Reacts Error boundry?

Reply:
> Is it not possible to use Reacts Error boundry?

 What do you mean?

Reply:
https://reactjs.org/docs/error-boundaries.html